documentation/docs/pages/operators/changelog.mdx

```diff
@@ -25,14 +25,276 @@ export const CiConfig = () => (
+
+**Operators taking part in Delegation program or Service Grant program must migrate their nodes latest by December 16th, 08:00 UTC.**
+</Callout>
+
+#### Updates
+
+- [Version count as a part of config score](tokenomics/mixnet-rewards#config-score-calculation) has been introduced. To familiarize yourself with Nym Node operator rewards calculation, read [this page](tokenomics/mixnet-rewards).
+- Nym nodes running as Exit Gateway in Service Grant program received delegation. Nym team is now delegating total of **64,800,000 NYM on top 241 Nym Nodes** (137 in Mixnode mode and 104 as Gateways). Our delegation aims to incentivise committed operators who support bootstrapping of Nym network before paying users come.
+
+<br />
+
+<AccordionTemplate name="Delegation size for Gateways in SGP">
+- 250k NYM - Upgrading to magura in time - 2 nodes
+- 300k NYM - Upgrading to magura + bonus for a quick patch upgrade - 102 nodes
+- No delegation - not upgrading in time - 2 nodes
+</AccordionTemplate>
+
+- `nym-node` has now implemented [IPv6 support for wireguard](https://github.com/nymtech/nym/pull/5059)
+
+- [`network_tunnel_manager.sh` updated](network): run the commands below to make sure 
+
+<br />
+
+<AccordionTemplate name={<TunnelManagerCommands/>}>
+These commands can be run one by one or copy-pasted and run as a block.
+```sh
+mkdir $HOME/nym-binaries; \
+
+curl -L https://raw.githubusercontent.com/nymtech/nym/refs/heads/develop/scripts/network_tunnel_manager.sh -o $HOME/nym-binaries/network_tunnel_manager.sh && \
+chmod +x $HOME/nym-binaries/network_tunnel_manager.sh; \
+
+$HOME/nym-binaries/network_tunnel_manager.sh check_nymtun_iptables  ; \
+$HOME/nym-binaries/network_tunnel_manager.sh remove_duplicate_rules nymtun0 ;\
+$HOME/nym-binaries/network_tunnel_manager.sh remove_duplicate_rules nymwg;\
+$HOME/nym-binaries/network_tunnel_manager.sh check_nymtun_iptables ; \
+$HOME/nym-binaries/network_tunnel_manager.sh adjust_ip_forwarding ; \
+$HOME/nym-binaries/network_tunnel_manager.sh apply_iptables_rules ; \
+$HOME/nym-binaries/network_tunnel_manager.sh check_nymtun_iptables ; \
+$HOME/nym-binaries/network_tunnel_manager.sh apply_iptables_rules_wg ; \
+$HOME/nym-binaries/network_tunnel_manager.sh configure_dns_and_icmp_wg ; \
+$HOME/nym-binaries/network_tunnel_manager.sh adjust_ip_forwarding ; \
+$HOME/nym-binaries/network_tunnel_manager.sh check_ipv6_ipv4_forwarding; \
+
+systemctl daemon-reload && service nym-node restart && journalctl -u nym-node -f 
+```
+
+Then run the jokes in a new window for control
+```sh
+$HOME/nym-binaries/network_tunnel_manager.sh joke_through_the_mixnet
+$HOME/nym-binaries/network_tunnel_manager.sh joke_through_wg_tunnel
+```
+</AccordionTemplate>
+
+#### Tools
+
+- **[New APIs documentation](../apis/introduction)** with interactive APIs generated from the OpenAPI specs of various API endpoints offered by bits of Nym infrastructure run both by Nym and community operators for both Mainnet and the Sandbox testnet.
```

Comment:
we're linking to a link that doesn't exist yet in this branch, but if we merge fast I suppose that's fine, vs changing the target of the OpenAPI doc PR ?

Reply:
I am aware, I won't deploy it up until all s merged together